docs/reference/using_typescript.md

```diff
@@ -38,12 +36,12 @@ export default class MyController extends Controller {
 }
 ```
 
-> The `declare` avoids override of the exist stimulus property and just define the type for TypeScript.
+> The `declare` avoids overriding the exist Stimulus property, and just defines the type for TypeScript.
```

Comment:
Should this be `existing`? Also might be worth to mention that `declare` is a keyword
```suggestion
> The `declare` keyword avoids overriding the existing Stimulus property, and just defines the type for TypeScript.
```